<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1277</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QVTKWidget" name="volumeRenderWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>611</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="colortf_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>561</width>
      <height>131</height>
     </rect>
    </property>
    <widget class="QLabel" name="colortf_min_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="colortf_max_label">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>110</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="colortf_bar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>551</width>
       <height>51</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="colortf_x_label">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="colortf_curbp_color_label">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>23</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="colortf_color_label">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Color:</string>
     </property>
    </widget>
    <widget class="QLabel" name="colortf_point_label">
     <property name="geometry">
      <rect>
       <x>271</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Point:</string>
     </property>
    </widget>
    <widget class="QScrollBar" name="colortf_verticalScrollBar">
     <property name="geometry">
      <rect>
       <x>218</x>
       <y>20</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="colortf_curbp_idx_label">
     <property name="geometry">
      <rect>
       <x>318</x>
       <y>29</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="colortf_curbp_x_label">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>72</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="opacitytf_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>561</width>
      <height>151</height>
     </rect>
    </property>
    <widget class="QFrame" name="opacitytf_bar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>551</width>
       <height>81</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QScrollBar" name="opacitytf_verticalScrollBar">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="opacitytf_point_label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Point:</string>
     </property>
    </widget>
    <widget class="QLabel" name="opacitytf_curbp_idx_label">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>29</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="opacitytf_opacity_label">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Opacity:</string>
     </property>
    </widget>
    <widget class="QLabel" name="opacitytf_curbp_opacity_label">
     <property name="geometry">
      <rect>
       <x>392</x>
       <y>28</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="opacitytf_x_label">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="opacitytf_curbp_x_label">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>30</y>
       <width>72</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="opacitytf_min_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>126</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="opacitytf_max_label">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>126</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>542</x>
       <y>111</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>542</x>
       <y>51</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1277</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <widget class="QMenu" name="menuSave_as">
     <property name="title">
      <string>Save as</string>
     </property>
     <addaction name="actionSaveAsSTL"/>
    </widget>
    <addaction name="actionOpenFolder"/>
    <addaction name="menuSave_as"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <widget class="QMenu" name="menuRenderStyles">
     <property name="title">
      <string>RenderStyles</string>
     </property>
     <addaction name="actionBoneStyle"/>
     <addaction name="actionBone2Style"/>
     <addaction name="actionSkinStyle"/>
     <addaction name="actionMuscleStyle"/>
    </widget>
    <widget class="QMenu" name="menuMapper">
     <property name="title">
      <string>Mapper</string>
     </property>
     <addaction name="actionRayCastMapper"/>
     <addaction name="actionSmartMapper"/>
    </widget>
    <addaction name="menuRenderStyles"/>
    <addaction name="actionBgColor"/>
    <addaction name="menuMapper"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenFolder">
   <property name="text">
    <string>OpenFolder</string>
   </property>
  </action>
  <action name="actionBgColor">
   <property name="text">
    <string>BgColor</string>
   </property>
  </action>
  <action name="actionBoneStyle">
   <property name="text">
    <string>Bone</string>
   </property>
  </action>
  <action name="actionBone2Style">
   <property name="text">
    <string>Bone2</string>
   </property>
  </action>
  <action name="actionSkinStyle">
   <property name="text">
    <string>Skin</string>
   </property>
  </action>
  <action name="actionMuscleStyle">
   <property name="text">
    <string>Muscle</string>
   </property>
  </action>
  <action name="actionRayCastMapper">
   <property name="text">
    <string>RayCastGPU</string>
   </property>
  </action>
  <action name="actionSmartMapper">
   <property name="text">
    <string>SmartMethod</string>
   </property>
  </action>
  <action name="actionSaveAsSTL">
   <property name="text">
    <string>STL</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="3" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
